package.json

```diff
@@ -28,20 +28,20 @@
   },
   "devDependencies": {
     "assert-diff": "^1.0.1",
-    "babel-cli": "^6.4.0",
+    "babel-cli": "^6.26.0",
     "babel-core": "^6.4.0",
     "babel-eslint": "^6.0.4",
     "babel-loader": "^6.2.1",
-    "babel-plugin-syntax-flow": "^6.3.13",
-    "babel-plugin-transform-flow-strip-types": "^6.4.0",
     "babel-preset-es2015": "^6.3.13",
+    "babel-preset-flow": "^6.23.0",
     "babel-preset-stage-1": "^6.3.13",
-    "babel-register": "^6.3.13",
+    "babel-babel-register": "^6.3.13",
```

Comment:
Typo - should be `babel-register`

Reply:
whoops! didn't even mean to touch that dep